test/test-applications/regression/NLog10LogsInjection.NullReferenceException/NLog10LogsInjection.NullReferenceException.csproj

```diff
@@ -1,8 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461</TargetFrameworks>
```

Comment:
Was there any reason to not run this on .NET Core on Windows?

Reply:
I assume [because NLog 1.0.0.5005](https://www.nuget.org/packages/NLog/1.0.0.505) doesn't support .NET Core

Reply:
Yeah, I expect it to crash at runtime, if not sooner, because it doesn't support .NET Core. Can you keep this as-is without the OS condition?

Reply:
Done in 1acaf20b.